.github/workflows/full_stack_tests.yml

```diff
@@ -164,6 +184,19 @@ jobs:
           path: /home/runner/work/oppia/oppia/core/tests/test-modules-mappings/acceptance/${{ steps.generate_suite_name_for_artifacts.outputs.MODIFIED_SUITE_NAME }}.txt
       - name: Run Mobile Acceptance Test ${{ matrix.suite.name }}
         run: xvfb-run -a --server-args="-screen 0, 1285x1000x24" python -m scripts.run_acceptance_tests --skip-build --suite=${{ matrix.suite.name }} --prod_env --mobile
+      - name: Run Mobile Acceptance Test ${{ matrix.suite.name }} (Attempt 2 with screen recording)
+        if: ${{ failure() }}
+        run: >
+          VIDEO_RECORDING_IS_ENABLED=1
+          xvfb-run -a --server-args="-screen 0, 1285x1000x24" 
```

Comment:
Are these the correct dimensions for a mobile viewport?

Reply:
I also doubted, but I had forgotten to ask. These dimensions have been used before. I will check it properly and reply.

Reply:
@seanlip dimensions in CI are for screen. Also, I found aspect ratio of test wasn't proper, so I fixed window size to 1920x1080 for all acceptance test instead of full screen. This will now only allow different viewports for mobile and desktop (like it was before) but this will ensure screen are recorded properly.

Also, do you want me to force fail one mobile acceptance test to check recordings?